app/helpers/layout_helper.rb

```diff
@@ -347,6 +347,18 @@ def fullscreen_button(element = "$(this).prev()")
   private
 
   def authorized_associations(associations)
-    associations.included_modules.include?(Authorizable) ? associations.authorized : associations
+    if associations.included_modules.include?(Authorizable)
+      if associations.respond_to?(:klass)
+        associations.authorized(authorized_assocations_permission_name(associations.klass), associations.klass)
+      else
+        associations.authorized(authorized_assocations_permission_name(associations), associations)
+      end
+    else
+      associations
+    end
+  end
+
+  def authorized_assocations_permission_name(klass)
+    "view_#{klass.to_s.downcase.pluralize}"
```

Comment:
> So what we could do is check whether a permission exists and return nil if not in authorized_assocations_permission_name. I suppose that would be better?

I'm in two minds about this and I'm not certain which way to go - perhaps even one approach for 1.9, another for 1.10.

1. A hard failure (exception) would be useful for, a) functional test that the taxonomy form displays as a non-admin user would immediately highlight any errors in core permissions, b) prevent the form and other resources being displayed to a non-admin if a plugin hasn't overridden the method.
2. Returning nil is good for compatibility, and means that plugins adding Taxonomix resources won't immediately start failing (bad just before a release), but means we won't know about any of the above conditions unless we log it loudly.